person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :age, :name
+
+  def of_age?
+    @age >= 18
+  end
+
+  private :of_age?
+
+  def can_use_services
+    of_age? || @parent_permission
+  end
```

Comment:
- [ ] [Optional] It would be better professionally if you put the private methods at the end of the class preceded with `private` like the following:
```ruby
def can_use_services
   of_age? || @parent_permission
end

private 

def of_age?
   @age >= 18
end
```